crates/astria-sequencer-types/src/sequencer_block_data.rs

```diff
@@ -265,6 +266,48 @@ impl SequencerBlockData {
         );
         Ok(())
     }
+
+    /// Get the height of the block.
+    #[must_use]
+    pub fn height(&self) -> Height {
+        self.header().height
+    }
+
+    /// Get the height of the block's parent.
+    ///
+    /// # Panics
+    ///
+    /// This function will panic if the block height is less than or equal to 1.
+    /// Only the genesis block has a height of 1, and all other blocks must have
+    /// a larger height.
+    #[must_use]
+    pub fn parent_height(&self) -> Height {
```

Comment:
i removed a few of these functions from SequencerBlockData. They were there originally because we weren't using SequencerBlockSubset yet and I forgot to removed them once that was added.